AwesomeTSProject/components/SingleCrypto/Styles.ts

```diff
@@ -0,0 +1,74 @@
+import styled from 'styled-components/native';
+
+export const OneCrypto = styled.View`
+  width: 100%;
+  height: 100px;
+  marginBottom: 10px;
+`;
+
+export const NamesContainer = styled.View`
+  width:25%;
+  position: relative;
+  left: 22%;
+  top: 20%;
+`;
+
+export const CryptoName = styled.Text`
+  color: black;
```

Comment:
create a theme file in your root, declare colors and use it for the colors

src/utils/theme.js

const theme = {
   colors: {
     black: 'black',
     white: '#fff'
     
   }
}